app/src/main/java/co/nimblehq/ui/main/MainActivity.kt

```diff
@@ -52,11 +52,11 @@ class MainActivity : BaseActivity() {
             .subscribe(this::gotoNextScreen)
             .bindForDisposable()
 
-        RxView.clicks(buttonRefresh)
+        buttonRefresh.clicks()
             .subscribe { viewModel.inputs.refresh() }
             .bindForDisposable()
 
-        RxView.clicks(buttonNext)
+        buttonNext.clicks()
```

Comment:
just note: We should rename the view using team convention.

Reply:
@manh-t What's the reason why we're not using `RxView` anymore? 🙏  Just curious

Reply:
@Tuubz we upgrade from `rxbinding2` to `rxbinding3` so the implementation has changed based on the version 😓